src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] The `App.js` component should be class-based. Please, fix it too. 🦊

Reply:
This issue is well addressed. Great work. :+1: :clap:

Reply:
Thank you for the feedback henatan